tests/unit/spec/base/mozilla-traffic-cop.js

```diff
@@ -40,6 +40,14 @@ describe('mozilla-traffic-cop.js', function () {
             cop.init();
             expect(cop.verifyConfig).toHaveBeenCalled();
         });
+
+        it('should initialize if DNT helper function is not available', function() {
+            var dntBak = window._dntEnabled;
+            window._dntEnabled = false;
```

Comment:
The issue is here we're trying to assert one thing, but actually testing something a bit different. You're correct though, using a stub would still return `true` for the funtion check. Just changing this line to `window._dntEnabled = undefined;` should work though, right?